scripts/caput-pipeline

```diff
@@ -131,23 +131,26 @@ mpirun -np %(mpiproc)i -npernode %(pernode)i -bind-to none python %(scriptpath)s
     slurm_script = """#!/bin/bash
 #SBATCH --account=%(account)s
 #SBATCH --nodes=%(nodes)i
-#SBATCH --ntasks-per-node=%(ppn)i
-#SBATCH --mem=%(mem)s
+#SBATCH --ntasks-per-node=%(ppn)i # number of MPI processes
```

Comment:
accodring to mpi scheduling it says:
Requesting --mem=0 is interpreted by Slurm to mean "reserve all the available memory on each node assigned to the job."
I kind of like that, what do you think?

Reply:
`ppn = ompnum * pernode` by default on SLURM, i.e. if you specify `ntasks-per-node` (`pernode`) and `cpus-per-task` (`ompnum`) then it will look for and allocate nodes that have that at least that number of cores (which was not the case on SciNet as jrs65 points out). So if you specify all three on slurm I'm guessing it will ignore one, or throw an error. I think we should have `ppn` not set by default on slurm.

Reply:
Yeah, I agree with Phil here. Also, I agree that setting `--mem=0` on slurm seems like the best default option.